src/deadline/unreal_adaptor/UnrealClient/step_handlers/unreal_render_step_handler.py

```diff
@@ -144,14 +155,41 @@ def create_queue_from_manifest(movie_pipeline_queue_subsystem, queue_manifest_pa
         :param movie_pipeline_queue_subsystem: The unreal.MoviePipelineQueueSubsystem instance
         :param queue_manifest_path: Path to the manifest file
         """
-        queue_manifest_path = queue_manifest_path.replace("\\", "/")
-        manifest_queue = unreal.MoviePipelineLibrary.load_manifest_file_from_string(
-            queue_manifest_path
-        )
+        manifest_path = queue_manifest_path.replace("\\", "/")
+        project_saved_dir = unreal.SystemLibrary.get_project_saved_directory().rstrip("/")
 
-        pipeline_queue = movie_pipeline_queue_subsystem.get_queue()
-        pipeline_queue.delete_all_jobs()
-        pipeline_queue.copy_from(manifest_queue)
+        # If QueueManifestPath under the Project's Saved dir (All stuff pulled via S3, all path mappings saved)
+        if manifest_path.startswith(project_saved_dir):
+            UnrealRenderStepHandler.copy_pipeline_queue_from_manifest_file(
+                movie_pipeline_queue_subsystem, manifest_path
+            )
+        # If Project synced via P4/UGS and queue manifest pulled via S3 to the OpenJob asset root
+        else:
+            serialized_manifest = unreal.MoviePipelineEditorLibrary.convert_manifest_file_to_string(
```

Comment:
Unfortunately no, because Unreal requires manifest file be located in project/Saved directory, in other case it will fail to load. So we just deserialize non-project maifest and save under the project folder

Reply:
Will try it